### ISSUE TITLE
Cannot add any attributes to an event due to "Invalid ISO 8601 format"

### ISSUE THREAD

**Issue by samitainio** [MISP/MISP#7461]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu 18.04.5 LTS
| OS version (client)       | Win10
| PHP version               | 7.2.24-0ubuntu0.18.04.7
| MISP version / git hash   | v2.4.143 (8766d0285ec60baab0a3b6bc76302640c7028d23)
| Browser                   | If applicable

### Expected behavior
I would be able to add attributes as usual.

### Actual behavior
![image](https://user-images.githubusercontent.com/5585477/120388212-7e9e8b00-c333-11eb-8044-1ae27d177986.png)

### Steps to reproduce the behavior
It worked before I updated the MISP and the Ubuntu packages to their latest available versions at the time.

After that I rebooted the server and noticed the symptom. There are no errors being generated to /var/www/MISP/app/tmp/logs/*.log files when clicking the Submit button in the UI. Only logs related to this symptom were found from the UI Logs -> List Logs which are listed below.

### Logs, screenshots, configuration dump, ...
```
1414320 	admin@admin.test 	ORGNAME 	2021-06-01 23:37:03 	Attribute 	0 	add 	Attribute dropped due to validation for Event 4023 failed: External analysis/comment Testing 	Validation errors: {"first_seen":["Invalid ISO 8601 format"],"last_seen":["Invalid ISO 8601 format"]} Full Attribute: {"event_id":"4023","category":"External analysis","type":"comment","distribution":"5","value":"Testing","comment":"","to_ids":0,"batch_import":"0","disable_correlation":0,"first_seen":"undefinedTundefined","last_seen":"undefinedTundefined","object_id":0} 
1414319 	admin@admin.test 	ORGNAME 	2021-06-01 23:36:54 	Event 	4023 	add 	Event (4023): Test event 	attribute_count () => (0), distribution () => (0), proposal_email_lock () => (0), locked () => (0), sighting_timestamp () => (0), disable_correlation () => (0), org_id () => (1), date () => (2021-06-01), threat_level_id () => (1), analysis () => (2), info () => (Test event), user_id () => (1), orgc_id () => (1), uuid () => (fd0619af-83e1-4ff9-98a9-ed181773173c) 
```

**Comment by iglocska** [comment]
Thanks for the heads-up, should be fixed now, just git pull again and it should work.

**Comment by samitainio** [comment]
@iglocska confirmed to be fixed in 3b2100e3d1e21c5c1d5e5cc7627bfe65278f910b. Thank you for the quick fix!
